### ISSUE TITLE
Format is non-mandatory, forces to jpg

### ISSUE THREAD

**Issue by alexbainbridge** [Ekito/ng2-cloudinary#12]
Hello

On get image, format is not mandatory

_"Deliver the file in the requested format by adding the required format as the file extension. The 'format' parameter itself is optional for use with Cloudinary's SDKs"_

However in https://github.com/Ekito/ng2-cloudinary/blob/master/src/cloudinary-image.service.ts there is 

`publicId + '.' + (transforms.format || 'jpg');`

Found this as I have some logos in PNG format that were coming out blurred as jpg - and couldn't work out how to get the image back in the format they were uploaded in. (my temporary workaround is to set that all come back in PNG !) 

Basically the getImageUrl function, if format is NOT set, should NOT add the . and not force to jpg

Thanks!

**Comment by julienboulay** [comment]
Hi @alexbainbridge,

jpg format might not be set by default. The format might be concatenated to the publicId, leaving it to the developer responsability.
But on the other side, when end users upload pictures to cloudinary, you might not know what is the format of the image and only retrieve the id of those images. Then it can be convenient to force to a default format like jpg.
The current solution allows to provide `format` attribute with cl-image component.
Is there any issue with that ?

**Comment by alexbainbridge** [comment]
Hi Julien

If format is unset , then the function defaults to jpg (which blurs png logos for example). The || makes that happen. 

What should happen is if format is unset, then the . and the jpg should not be added. i.e. the publicID will be the last value in the generated URL

When there is no filetype extension cloudinary gives you back the image in the format it was uploaded in.

**Comment by alexbainbridge** [comment]
I created a pull request with a fix for this

https://github.com/Ekito/ng2-cloudinary/pull/13

**Comment by julienboulay** [comment]
Fixed by PR #13 
Thanks

**Comment by alexbainbridge** [comment]
Great!

Incidentally, I did consider adding a new test - as really you should have one for when format is set.... e.g. format="jpg" should add .jpg to the URL.
